ducktape/cluster/remoteaccount.py

```diff
@@ -39,6 +40,20 @@ def __str__(self):
         r += self.hostname
         return r
 
+    def __repr__(self):
+        return str(self.__dict__)
+
+    def __eq__(self, other):
```

Comment:
No, but you can sort them if you need that. Your current implementation doesn't guarantee ordering either, does it? `keys()` has the same ordering limitations as `items()`.